src/server/services/generate-arpa-report.js

```diff
@@ -125,42 +116,53 @@ async function generateReport (periodId) {
     { name: 'project51518BulkUpload', func: generateProject51518 },
     { name: 'project519521BulkUpload', func: generateProject519521 },
     { name: 'projectBaselineBulkUpload', func: generateProjectBaseline },
-    { name: 'expendituresGT50000BulkUpload', func: generateExpendituresGT50000 },
-    { name: 'expendituresLT50000BulkUpload', func: generateExpendituresLT50000 },
-    { name: 'paymentsIndividualsLT50000BulkUpload', func: generatePaymentsIndividualsLT50000 },
+    {
```

Comment:
do you have a max line length setting in your editor? we should agree on max line length, i don't think one is enforced right now.

Reply:
Yeah, good question.

I don't have a strong personal preference here.  What do you think about adopting prettier checks in CI and removing conflicting eslint rules to resolve this?  Is prettier auto-formatting easy to work with your IDE?